### PR TITLE
Remove 'ssl_' suffix from the package name

### PR DESCRIPTION
**Pull request by alinbalutoiu**

### CODE REVIEW

**Review by jeffaco — CHANGES_REQUESTED** [review]
I'm remote right now, and it is hard to tell if you have a if you have covered both RPM and DEB packages. Does this change cover both?

**Comment by alinbalutoiu** [comment]
@jeffaco thanks for pointing that out, I created a new variable for the package name since it has the same format for both package types.

**Review by jeffaco — CHANGES_REQUESTED** [review]
Close. But when building Debian vs. RPM packages, we generally have a (very) short DEB/RPM specific installbuilder file for dependencies. Given that the omi-script-provider has a dependency on OMI, it certainly makes sense that the package calls for that.

Please look at SCXCore for an example of how this is done, let me know if you have questions.

**Comment by alinbalutoiu** [comment]
I added a dependency on OMI which will require a minimum version of 1.3.0.99, that is the first OMI build which contains the support for the Python OMI script provider.

File: installbuilder/Linux_RPM.data [hunk 1]
```diff
@@ -0,0 +1,5 @@
+%Variables
+PACKAGE_TYPE: 'RPM'
+
+%Dependencies
+omi (>= 1.3.0.99)
```
Comment (jeffaco) [inline installbuilder/Linux_RPM.data]: I don't believe this is correct syntax for RPM. Did you test this with an old version of OMI installed for the RPM case?

See [SCXCore](https://github.com/Microsoft/SCXcore/blob/38d4949059b0e21ee072d9f83353acd2d7268753/installer/datafiles/Linux_RPM.data#L6) for more information.

**Review by jeffaco — CHANGES_REQUESTED** [review]
I don't believe your syntax is correct for RPM files, see comments.

File: installbuilder/Linux_RPM.data (hunk 1 above)
Comment (alinbalutoiu) [reply installbuilder/Linux_RPM.data]: Seems to be a bit confusing, for DPKG the version is 1.3.0.99 and for RPM the version is 1.3.0-99. Thanks for pointing that out. PR updated.

**Review by jeffaco — APPROVED** [review]
Looks good to me!

**Pull request merged**